docs/player-support/ha.md

```diff
@@ -18,4 +24,4 @@ Music Assistant has support for playing to media player entities in Home Assista
 - Synchronisation between this player type and any others is not possible
 - In order to support a greater number of players, different streaming profiles are available. If the player doesn't work, stops mid stream or has other playback issues then change the player setting `HTTP Profile used for sending audio` and try each option until the player works
 - If there is no metadata sent to the player then you can trying enabling the option `Try to ingest metadata into stream`
-- ESPHome based Media Players are in general not recommended for playback of music. Short audio announcements and maybe even webradio could work but it is really not suitable for playing music from Music Assistant. TIP: you may have to enable the "fixed content length" HTTP profile in the player's settings. The ESPHome is working on an improved media player which can be used on ESP32 modules with PSRAM, which will work correctly with Music Assistant.  
+- ESPHome based Media Players are in general not recommended for playback of music. Short audio announcements or possibly webradio could work but it is really not suitable for playing music from MA. TIP: you may have to enable the "fixed content length" HTTP profile in the player's settings. The ESPHome project is working on an improved media player which can be used with ESP32 modules with PSRAM and, when available, these will work correctly with MA
```

Comment:
This new media player is available in Voice PE and some others: https://discord.com/channels/753947050995089438/1324479356126367865

Reply:
Fixed I think